### ISSUE TITLE
DeepSite binding pocket prediction

### ISSUE THREAD

**Issue by rbharath** [deepchem/deepchem#585]
There's been some good work [recently](https://academic.oup.com/bioinformatics/article-abstract/doi/10.1093/bioinformatics/btx350/3859178/DeepSite-Protein-binding-site-predictor-using-3D?redirectedFrom=fulltext) on deep learning for binding site prediction. DeepChem has some rudimentary binding pocket prediction methods already, but it would be great to get an implementation of DeepSite into DeepChem.

**Comment by levipierce** [comment]
Yeah I like this idea.  I did quite a bit of hacking on fpocket recently might be useful.  I will check out the paper...

**Comment by rbharath** [comment]
PRs welcome! :-)

**Comment by patrickhop** [comment]
this would be awesome

**Comment by JimSEOW** [comment]
@rbharath there are many pocket prediction tools. As we proceed towards polypharmacology, we need to explore the wobbly upper part of the 3D pocket. 

Also, how good is deepChem is classifying pocket at invariant 3D shape level?

**Comment by rbharath** [comment]
Not clear. The binding pocket prediction in DeepChem is very untested right now. Invite comments on performance of various pocket prediction methods on this thread :)
